pypeit/par/pypeitpar.py

```diff
@@ -1549,13 +1550,18 @@ def __init__(self, only_slits=None, exclude_slits=None, offsets=None, spat_toler
                             'the weights only if ``weights = auto``.'
         # TODO For echelle coadds this should just default to 1
 
+        # TODO: Why is this spat:spec and not spec:spat like everything else??
         # manual extraction
         defaults['manual'] = None
         dtypes['manual'] = str
```

Comment:
```suggestion
        dtypes['manual'] = [str, list]
```

I am adding a test in the Dev suite for manual+mosaic (see Dev Suite branch), but I notice it was failing. This change will fix it.

Reply:
Fixed!